src/client/features/search/index.js

```diff
@@ -78,9 +76,10 @@ class Search extends React.Component {
     this.setState({
       landingLoading: true
     },()=>{
-      ServerAPI.findUniprotId(query).then(res=>{
+      ServerAPI.geneQuery(query).then(res=>{
+        res= res.geneInfo.map(gene=>gene.convertedAlias);
```

Comment:
Might want to consider renaming `res` and `result` to clearer names since they're used in the same place.